rosetta/cosmos/launchpad/network.go

```diff
@@ -17,8 +23,45 @@ func (l Launchpad) NetworkList(ctx context.Context, request *types.MetadataReque
 	}, nil
 }
 
+type nodeInfo struct {
+	Version string `json:"version"`
+}
+
+type nodeResponse struct {
+	NodeInfo nodeInfo `json:"node_info"`
+}
+
 func (l Launchpad) NetworkOptions(ctx context.Context, request *types.NetworkRequest) (*types.NetworkOptionsResponse, *types.Error) {
-	panic("implement me")
+	addr := l.endpoint + endpointNodeInfo
+	resp, err := l.c.Get(addr)
+	if err != nil {
+		return nil, rosetta.NewError(1, "error getting data from node")
+	}
+	defer resp.Body.Close()
+
+	var nodeResp nodeResponse
+	if err := json.NewDecoder(resp.Body).Decode(&nodeResp); err != nil {
+		return nil, ErrInterpreting
+	}
+
+	return &types.NetworkOptionsResponse{
+		Version: &types.Version{
+			RosettaVersion: "1.2.5",
+			NodeVersion:    nodeResp.NodeInfo.Version,
+		},
+		Allow: &types.Allow{
+			OperationStatuses: []*types.OperationStatus{
+				{
+					Status:     "SUCCESS",
+					Successful: true,
+				},
+			},
+			OperationTypes: []string{
+				"Transfer",
```

Comment:
Interesting this, I would like to see it related to the config we put on the Network struct we created for the Server.

Reply:
done by 823e030